openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/Shared/Badge.less

```diff
@@ -0,0 +1,17 @@
+.announcement-badge {
+  width: 24px;
+  height: 24px;
+  background: #ffc143;
+  border: 1px solid #ffc143;
```

Comment:
Since we are already having css file can we less dependent on tailwind?

Reply:
Yes @chirag-madlani, I will try to eliminate tailwind classes.